masq/tests/utils.rs

```diff
@@ -152,3 +161,59 @@ fn child_from_command(command: &mut Command) -> Child {
         .spawn()
         .unwrap()
 }
+
```

Comment:
decide whether we should preserve this code, maybe there is not a real use case for it